package.json

```diff
@@ -14,34 +14,34 @@
     "access": "public"
   },
   "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.3",
-    "babel-jest": "^23.6.0",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-latest": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1",
+    "@babel/core": "^7.10.2",
+    "@babel/preset-env": "^7.10.2",
+    "@babel/preset-typescript": "^7.10.1",
+    "@babel/types": "^7.10.2",
+    "@rollup/plugin-babel": "^5.0.2",
+    "@rollup/plugin-node-resolve": "^8.0.0",
+    "@types/jest": "^25.2.3",
+    "babel-eslint": "^10.1.0",
     "eslint": "^4.19.1",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-security": "^1.4.0",
-    "jest": "^23.6.0",
+    "jest": "^26.0.1",
     "np": "^3.1.0",
     "prettier": "^1.13.3",
-    "rollup": "^0.59.2",
-    "rollup-plugin-babel": "^3.0.4",
+    "rollup": "^2.12.1",
     "rollup-plugin-commonjs": "^9.1.3",
     "rollup-plugin-eslint": "^4.0.0",
-    "rollup-plugin-node-resolve": "^3.3.0",
     "rollup-plugin-replace": "^2.0.0",
-    "rollup-plugin-uglify": "^4.0.0"
+    "rollup-plugin-uglify": "^4.0.0",
+    "typescript": "^3.9.3"
   },
   "scripts": {
     "build": "rollup -c",
+    "check-types": "tsc",
```

Comment:
Would you please change `check-types` to

```json
"compile": "tsc --pretty",
"compile:watch": "tsc --pretty --watch",
```

That's what we (mostly) use in other repos.

Reply:
If you wanted to type one file, utils.ts would be the most straightforward. But I like the idea of just getting TS in place and then converting them later on, or as we touch those files.

Reply:
> If you wanted to type one file, utils.ts would be the most straightforward. But I like the idea of just getting TS in place and then converting them later on, or as we touch those files.

I do have that one typed locally already. But yeah, since there were so many changes just to the build process, I figured leaving out the actual source code changes would be better for clarity.